### PR TITLE
1502 docking persistence

### PR DESCRIPTION
**Pull request by giduac**
1502 docking persistence
[issue 1502](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1502)
Removes the issues found.
Adds a form and button to TestForm for workspace testing.

compile-results

### CODE REVIEW

**Review by Wagnerp — CHANGES_REQUESTED** [review]
See notes

**Comment by Smurf-IV** [comment]
> See notes

@Wagnerp I Cannot see them ?? Did you mark it as Needs Review (Or similar)

**Review by Wagnerp — CHANGES_REQUESTED** [review]
See notes

**Comment by giduac** [comment]
> See notes
@Wagnerp 
Which notes do you mean. Don't see any here, or am I missing something?

**Comment by Smurf-IV** [comment]
> See notes

nope..
image

**Comment by Wagnerp** [comment]
> > See notes
> 
> @Wagnerp I Cannot see them ?? Did you mark it as Needs Review (Or similar)

@Smurf-IV The GitHub mobile app is not very easy to use, but the changelog needs adjustment

File: Documents/Changelog/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 =======
 
 ## 2024-11-xx - Build 2411 - November 2024
+* Resolved [#1388](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1502) Fixes some problems creating workspaces introduced through warnings removal.
```
Comment (Smurf-IV) [inline Documents/Changelog/Changelog.md:6]: Some comment, 
i.e. the Issue number in the link needs to be modified as well

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

**Comment by Smurf-IV** [comment]
This,
image

Then when completed, These
image

**Comment by Wagnerp** [comment]
> This,
> image
> 
> Then when completed, These
> image
> 

Yes, it's easier to do it on a proper PC!

File: Documents/Changelog/Changelog.md (hunk 1 above)
Comment (giduac) [reply Documents/Changelog/Changelog.md:6]: Good catch (copy paste....)

Comment (giduac) [reply Documents/Changelog/Changelog.md]: @Wagnerp 
And corrected.

Comment (Wagnerp) [reply Documents/Changelog/Changelog.md]: @giduac Sorry for the confusion, I am away at the moment and relying on the GitHub app, but the usability of it is a bit weak

Comment (giduac) [reply Documents/Changelog/Changelog.md]: NP, should be OK now.
@Smurf-IV can merge/take care, if he's happy with it...

**Pull request merged**